worker/src/bin/worker.rs

```diff
@@ -27,13 +27,9 @@ struct Args {
     #[clap(long)]
     prometheus: Option<url::Url>,
 
-    /// Port to use for diagnostics web server
+    /// Port this worker instance can be reached on by runner & diagnostics web server
     port: Option<u16>,
```

Comment:
This should be an option, not an argument.

Reply:
Gotcha!

REF: 54eda8f460ca14e6809e4cc291f07bec9bde54f7